stable/external-dns/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: Configure external DNS servers (AWS Route53, Google CloudDNS and others)
   for Kubernetes Ingresses and Services
 name: external-dns
-version: 0.6.1
-appVersion: 0.5.0
+version: 0.6.2
+appVersion: 0.5.3
```

Comment:
`0.5.3` has a bug 😞  let's wait for `0.5.4` or go with `0.5.2`.

Reply:
Agreed. We don't need this node list permission until 0.5.3, but we should wait for 0.5.4 to get the fixes for the cache bugs that got introduced in 0.5.3.

Reply:
:+1:  just tried 0.5.3 and hit bugs

Reply:
We released 0.5.4

Reply:
https://github.com/kubernetes-incubator/external-dns/releases/tag/v0.5.4